### ISSUE TITLE
Option to remove things not focused on the main core aspect of the game

### ISSUE THREAD

**Issue by LaCumbiaDelCoronavirus** [studio-minus/ppg-bugs-and-requests#235]
**Feature description**
Remove a few but not all "spoooooky" aspects of the game. This would be a setting in the settings menu, when turned on,
would basically remove alot of the scary aspects in the game such as: large bodies of water being darker the deeper they go, just making it lighter but not to remove all the detail to make the game look bad visually.

**Feature purpose**
Option to focus back on the main core of the game; killing humans and making things and removing things like that **HORROR** and yes *i know you can manually disable ambient in audio* but this is for removing such things as:
Any type of scary stuff, the scary ambient too. But this option is not to go so far as to remove little details such as fans in Snow or the pipes in Sea.

***TL;DR,*** option remove scary thing

**Additional context**

**Comment by idotwaffle** [comment]
be more specific on what's "scary"
is the pumpkin spirit scary? are gorses scary? are taxes scary?

**Comment by LaCumbiaDelCoronavirus** [comment]
> be more specific on what's "scary" is the pumpkin spirit scary? are gorses scary? are taxes scary?
> 
> i don't find the water getting darker scary, it just makes sense. if you don't like how dark it is, just don't go down there a few ambient noises are the only thing i'd possibly call scary so maybe those should be togglable

Pumpkin spirit or gorses arent scary, ill explain further right here on what i consider """scary""" thing that intermingle with gameplay.
Darkness in any body of water doesnt let you see things in it clearer or that tunnel in hybrid. I only want this setting to possibly remove some of that "unsettling" vibe from the game. Also the sound in humongous, multiple secrets in Blocks, ambients from tower and Abyss.  Things that i do know about but i and other people dont consider as scary includes harmless secrets like fans in Snow, keypad in Abyss, those pipes (i think) that may or may not still be in Sea, all that might be enough to make a spectrum 
to visualize. Though about the water getting darker, i think that it gets too dark and with the fact that this game has many, many secrets.
